bazel/revisions.bzl

```diff
@@ -6,12 +6,14 @@ EMSCRIPTEN_TAGS = {
         hash = "2ddc66235392b37e5b33477fd86cbe01a14b8aa2",
         sha_linux = "8b840819eb88f9178c11bad25859ce448a0559e485823a863a6add21380636ca",
         sha_mac = "0cb3f9bfbcc744233eae9d20036155738409405eacf8a3d4f9beefc5919d809a",
+        sha_mac_arm64 = "ee2772f380419df17d154e00388a16bcddc78c7af035c16a2ee534d6ecf099aa",
         sha_win = "c0549e1dbaa581ae66934c38beebd4250cd450cc2778e9a602cd9431bc81bc37",
     ),
     "3.1.2": struct(
         hash = "6626e25d6d866cf283147ca68d54ac9326fe399f",
         sha_linux = "4fb53364a2ba1de8978445aa26b2204bfd215b41da5d7df04f231040b197010a",
         sha_mac = "a8e347accb1ff402d96a128912ac8cda1731611c9f89095fee0ad39a6a18bbc3",
+        sha_mac_arm64 = "4374f5c852d0403b0a3b0e9dc8a3856a340e9d82ecf0f20aa8b36c6179d31fc8",
```

Comment:
Is it possible to backfill some of these older versions between 3.1.2 and 2.0.33?

Reply:
Most likely there were never actually built by the builders.. and building them would be a fair amount of work for someone.

We generally really on the waterfall to produce all our builds for us: https://ci.chromium.org/p/emscripten-releases

Sine x64 binaryen happen to run fine on M1 silicon (albient slightly slower), one possible solution would be to simply copy the `sha_mac` for all the older releases.     Can you confirm that this works?   And if so maybe you could open a PR to duplicate the `sha_mac` lines for all older releases?

Reply:
I don't know if it will be that simple, I think we'd have to tell bazel to use the x86 arch. I also wonder if we could refactor the code a bit so that it doesn't fail on the missing arm links. The issue I am seeing is if I use a new emsdk version but try to force an old emscripten version, it fails saying the arm url is not defined, even though I'm not building for arm

Reply:
I'm not sure what you mean can you just point `sha_mac_arm64` at the tar archive for `sha_mac`?  Why wouldn't that work.  The x86 executables should run completely transparently on M1, right?

Reply:
Wait.. are you not running on M1 hardware?   The `sha_mac_arm64` archive is for use on M1 hardward.. it has nothing to do with targeting arm (emscripten doesn't target arm or x86 .. only wasm)

Reply:
I'm aware it's for M1 only. I am not running on M1. I am simply trying to use older versions from newer emsdk releases and they fail because the arm url is not defined. Copying the URLs might work, I just don't have a machine to try it. If it doesn't work, we should be able to shuffle the bazel scripts to avoid failing because arm URLs aren't defined, if I am just building on x86

Reply:
Sure... the missing arm build should not block installation on non-arm.  Can you open a new bug about that?